### ISSUE TITLE
[FEATURE] view node links

### ISSUE THREAD

**Issue by nil0x42** [giuspen/cherrytree#249]
i think it will be very usefull to be able to list nodes which link to selected node.

more important even:
    when trying to delete a node, receive an alert if other nodes have link to current node, which will then be broken, and list them so we can cancel to sanitize concerned nodes if we want.

**Comment by philippHRO** [comment]
Yes, this is absolutely needed. I remember myself wanting this a few times in the past, when I was cleaning up my cherry tree document.
